src/components/CytoscapeGraph/graphs/ConcentricGraph.ts

```diff
@@ -1,8 +1,7 @@
-export class CoseGraph {
+export class ConcentricGraph {
   static getLayout() {
     return {
-      name: 'cose-bilkent',
-      animate: false,
```

Comment:
I know 'animate' defaults to false, but I'm not sure it's a bad idea to make our intent explicit.   Not a blocker.

Reply:
+1 I'll add it.